File: java/sprint1_nonfinals/E.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;


public class E {

    private static String getLongestWord(String text) {
        // Ваше решение
    }
    public static void main(String[] args) throws IOException {
        try (BufferedReader reader = new BufferedReader(new InputStreamReader(System.in))) {
            int textLength = readInt(reader);
            String text = reader.readLine();
            String longestWord = getLongestWord(text);
            System.out.println(longestWord);
            System.out.println(longestWord.length());
        }

    }

    private static int readInt(BufferedReader reader) throws IOException {
        return Integer.parseInt(reader.readLine());
    }

}

File: csharp/sprint1_nonfinals/E.cs
using System;
using System.IO;

public class E
{
    private static TextReader reader;
    private static TextWriter writer;

    private static string GetLongestWord(string text)
    {
        // Ваше решение
    }

    public static void Main(string[] args)
    {
        reader = new StreamReader(Console.OpenStandardInput());
        writer = new StreamWriter(Console.OpenStandardOutput());

        var textLength = ReadInt();
        var text = reader.ReadLine();
        var longestWord = GetLongestWord(text);

        writer.WriteLine(longestWord);
        writer.WriteLine(longestWord.Length);

        reader.Close();
        writer.Close();
    }

    private static int ReadInt()
    {
        return int.Parse(reader.ReadLine());
    }
}
